bridge/shared/models/trace.ts

```diff
@@ -16,8 +16,10 @@ export interface IEvaluationData {
   sloFileContent: string;
   timeEnd: Date;
   timeStart: Date;
-  score_pass?: string;
-  score_warning?: string;
+  score_pass: number;
```

Comment:
Actually, this property does not exist initially. It's not provided by the API but by the bridge client.
Same for `score_warning`, `score_info` and `score_state`. Could you mark it as optional?